toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -229,6 +240,20 @@ func (r *RpmRepoCloner) initializeRepoDefinitions(repoDefinitions []string) (err
 		}
 	}
 
+	// Add each previously existing repofile to the end of the new file, then delete the original.
+	// We want to try our custom mounted repos before reaching out to the upstream servers.
+	for _, repoFile := range existingRepoFiles {
```

Comment:
I suggest filtering the `existingRepoFiles` to files that end with `.repo` only.

Reply:
Done, using filepath.Glob() now